server/vertx/src/main/scala/sttp/tapir/server/vertx/encoders/package.scala

```diff
@@ -0,0 +1,24 @@
+package sttp.tapir.server.vertx
+
+import java.io.InputStream
+
+import io.vertx.core.buffer.Buffer
+
+package object encoders {
+
+  private val bufferSize = 1024
+
+  /**
+   * README: Tests are using a ByteArrayInputStream, which is totally fine,
```

Comment:
I think we should always assume that an `InputStream` will be blocking - in both request&response contexts

Reply:
Ok, so I'll wrap inputstream within ` executeBlocking` (and change signatures to return `Future`s everywhere)